### PR TITLE
docs: configuration java demo (#687)

### LINKED ISSUE #687: Run configuration API quickstart with java sdk; 在 configuration API quickstart 中添加 java demo
**What would you like to be added**:
Currently users can run Layotto state API & sequencer API quickstart with Java sdk, which is friendly for Java developers who don't have golang environment on their laptop. See https://mosn.io/layotto/#/en/start/state/start
and
https://mosn.io/layotto/#/zh/start/sequencer/start

image

Next, we need to support running [configuration API quickstart](https://mosn.io/layotto/#/zh/start/configuration/start) with java sdk.

**Why is this needed**:
#67

### PR DESCRIPTION
**Pull request by it-linnan**
**What this PR does**:

Add configuration java demo

**Which issue(s) this PR fixes**:

Fixes #687

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by seeflood** [comment]
Thanks! @it-linnan 
Please sign the CLA first

**Comment by it-linnan** [comment]
> Thanks! @it-linnan Please sign the CLA first

got it

File: docs/zh/start/configuration/start-apollo.md [hunk 1]
```diff
@@ -52,6 +52,10 @@ go build -o layotto
 > 遇到这种情况，您可以先尝试其他 demo，例如 [etcd demo](zh/start/configuration/start)
 
 ## 第三步：启动客户端Demo，调用Layotto增删改查
```
Comment (seeflood) [inline docs/zh/start/configuration/start-apollo.md:54]: Could u modify the english doc too?
https://mosn.io/layotto/#/en/start/configuration/start-apollo
https://mosn.io/layotto/#/en/start/configuration/start

**Review by seeflood — CHANGES_REQUESTED** [review]

File: docs/zh/start/configuration/start-apollo.md (hunk 1 above)
Comment (it-linnan) [reply docs/zh/start/configuration/start-apollo.md:54]: Ok!

**Comment by seeflood** [comment]
@it-linnan Hi, l left a comment in java-sdk repo, https://github.com/layotto/java-sdk/pull/50#discussion_r996453407 
Please fix the java-sdk first. Thanks!

**Comment by it-linnan** [comment]
> @it-linnan Hi, l left a comment in java-sdk repo, [layotto/java-sdk#50 (comment)](https://github.com/layotto/java-sdk/pull/50#discussion_r996453407) Please fix the java-sdk first. Thanks!

I'll fix it as soon as possible.

**Comment by seeflood** [comment]
@zhenjunMa Hi, could u help review the java sdk PR https://github.com/layotto/java-sdk/pull/50

**Pull request closed without merging**